adoc/attributes.adoc

```diff
@@ -25,11 +25,11 @@
 :base_os_version: 15 SP1
 :cilium_version: 1.5.3
 :envoy_version:
-:crio_version: 1.16.0
-:dex_version: 2.16.0
-:etcd_version: 3.3.15
+:crio_version: 1.17.0
+:dex_version: 2.32.0
+:etcd_version: 3.4.3
 :gangway_version: 3.1.0-rev4
-:kube_version: 1.16.2
+:kube_version: 1.18.0
 :kubedoc: https://v1-16.docs.kubernetes.io/docs/
 :kured_version: 1.2.0-rev4
```

Comment:
kured is also at 1.3.0, and will probably be bumped to 1.4.0.

Reply:
Kubernetes is also 1.18.2, IIRC.

Reply:
done